app/views/likes/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>This is posts#new page</h1>
+
+<%= form_for :like, url: user_post_likes_path(@user, @post) do |form|%>
+    <div>
+        <%= form.submit %>
+    </div>
+<%end%>
```

Comment:
- [ ] **[OPTIONAL]** Instead of creating an entirely new page for liking a post, we can just add a like button on the post show page and `create` the like there without visiting a new like page. Just like facebook, or other social media platforms. This would mean the `new` like route and controller action would not be needed, only the `create` route/action